oap-server/server-storage-plugin/storage-iotdb-plugin/pom.xml

```diff
@@ -49,6 +49,14 @@
                     <groupId>net.jpountz.lz4</groupId>
                     <artifactId>lz4</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
```suggestion
               <!-- logback has runtime negative impact about performance, -->
               <!-- Disable this, and use slf4j-sl4j instead of. -->
                <exclusion>
```